core/src/Kokkos_CopyViews.hpp

```diff
@@ -3492,19 +3496,14 @@ create_mirror(const Kokkos::View<T, P...>& src,
       "The view constructor arguments passed to Kokkos::create_mirror must "
       "not explicitly allow padding!");
 
+  if (src.impl_is_strided_from_padding()) {
+    auto prop_copy = Impl::with_properties_if_unset(
+        arg_prop, std::string(src.label()).append("_mirror"), AllowPadding);
```

Comment:
Could you factor out the common expression for the label string?

Reply:
Yes, I'll force push